### DISCUSSION TITLE
Probe returning response of zero bytes

### DISCUSSION THREAD

**Discussion by snyderra** [belaban/JGroups#849] [Q&A]
I have 3 android devices connected via multicast (udp.xml) and then run the probe from another computer on the same network.

> java -cp jgroups-5.3.13.Final.jar org.jgroups.tests.Probe -mcast_addr 224.0.75.75 -port 7500
> #1 (215 bytes):
> BD26 class com.example.EndpointId:05167963485402A5 /10.9.8.111 [ip=10.9.8.111:58142, 1 mbr(s), cluster=0000DEADBEEF0000, version=5.3.13.Final-routerfix+setcache (Zoncolan) (java 2.1.0)]
> 
> #2 (215 bytes):
> BD46 class comexample.EndpointId:05167962BC31B6EE /10.9.8.100 [ip=10.9.8.100:35425, 1 mbr(s), cluster=0000DEADBEEF0000, version=5.3.13.Final-routerfix+setcache (Zoncolan) (java 2.1.0)]
> 
> #3 (0 bytes):
> 
> 3 responses (3 matches, 0 non matches)

The 3 android devices are communicating as expected. What would cause the third response to be 0 bytes?
The only changes to JGroups are from pull requests that have already been merged. The probe is running from an unmodified jar

Any ideas what would cause this?

**Comment by belaban** [comment]
Note that your cluster hasn't formed: it shows 1 member in each of the 2 valid responses.
As to the 0 response, I don't know. I suggest:
* Debug this, both on the server (DiagnosticsHandler) and the client (Probe)
* Use wireshark to look at the packets (is the 0 response always coming from the same member?)
* Increase the timeout to see if this helps (probe.sh -timeout X)
* Use netcat to access the members directly: nc -u <IP address> 7500
